src/parser.rs

```diff
@@ -142,6 +152,20 @@ impl Parser {
         Ok(i)
     }
 
+    fn parse_optional_param(&self, chunk: Option<&&str>) -> Result<Option<Param>, ParseError> {
+        match chunk {
+            None => Ok(None),
+            Some(chunk) => {
+                if *chunk == "_" {
+                    return Ok(None);
+                }
```

Comment:
Should not be necessary, but if there are some calls that omits a middle argument then we can write 
```
sys 0 '20' '2' _ '3'
```

to omit the 2nd argument